contentcuration/contentcuration/views/zip.py

```diff
@@ -99,4 +117,9 @@ def get(self, request, zipped_filename, embedded_filepath):
         # allow all origins so that content can be read from within zips within sandboxed iframes
         response["Access-Control-Allow-Origin"] = "*"
```

Comment:
This can be replaced with `_add_access_control_headers(request, response)` as done here, unless there was some reason not to: https://github.com/learningequality/kolibri/pull/3707/files#diff-779e399ad9c5c5db74338f065af5abbeL75

Reply:
My Very Good Reason is that is that I missed it, and since it looked like the right choice my brain didn't flag it during review. ;-) In short, thanks for catching this, update incoming!